src/pages/Room/index.js

```diff
@@ -33,29 +33,17 @@ import { rooms } from 'mocks';
 
 const Room = () => {
   const [error, setError] = useState('');
-  const [data, setData] = useState({});
+  const [buttonImage, setButtonImage] = useState();
   const [loading, setLoading] = useState(false);
   const { id } = useParams();
-
   const room = find(rooms, { id });
 
   const fetchConfig = () => {
     setLoading(true);
-    fetch('https://staging-api-pay.fanz.io/v1/payments/config', {
-      method: 'get',
-      headers: {
-        Authorization:
-          '0653d9f4cdff4de5edbb6b22cd2d5f3b:6b0436eca2e69575d638f13d216c61579849ecb114f41ff836497f58e2eea069'
-      }
-    })
-      .then(function(response) {
-        if (!response.ok) {
-          throw response;
-        }
-        return response.json();
-      })
-      .then(function(data) {
-        setData({ ...data.data });
+    const clientId = 'cus-MTYyNzY0ODkzMTE0NGszdXlt';
```

Comment:
Can you move this out of this file. maybe create it on the `constant` or sth not hard code like this

Reply:
fixed